android/src/main/java/ee/forgr/capacitor_updater/CapacitorUpdater.java

```diff
@@ -333,6 +335,26 @@ public void onReceive(Context context, Intent intent) {
     }
   };
 
+  private String decryptChecksum(String checksum, String version)
```

Comment:
what is this function?

Reply:
as we spoked is the second factor to make the encryption harder to break